newIDE/app/src/ResourcesList/ResourcePreview/ImagePreview.js

```diff
@@ -147,6 +152,15 @@ const ImagePreview = (props: Props) => {
     }));
   };
 
+  const resourceIsSmooth = (
```

Comment:
Put this outside of the component function, to avoid re-allocating a function everytime while it's not needed (this function is "pure", it takes only its arguments to do its job. So you can make it "free", outside of the component, and the JS engine will only need to initialize it once).